pyvista/plotting/plotting.py

```diff
@@ -2957,31 +2957,59 @@ def add_text(self, text, position='upper_left', font_size=18, color=None,
         self.add_actor(self.textActor, reset_camera=False, name=name, pickable=False)
         return self.textActor
 
-    def open_movie(self, filename, framerate=24):
+    def open_movie(self, filename, framerate=24, quality=5, **kwargs):
         """Establish a connection to the ffmpeg writer.
 
         Parameters
         ----------
         filename : str
             Filename of the movie to open.  Filename should end in mp4,
-            but other filetypes may be supported.  See "imagio.get_writer"
+            but other filetypes may be supported.  See ``imagio.get_writer``
 
         framerate : int, optional
             Frames per second.
 
+        quality : int, optional
+            Quality 10 is the top possible quality for any codec. The
+            range is ``0 - 10``.  Higher quality leads to a larger file.
```

Comment:
Can we add the test for `quality` ?

Reply:
We could add a test to see if the quality changes, I don't think that makes sense since we're always passing quality to `get_writer`, and adding an additional test wouldn't improve coverage.  I'd rather just pass the variable and assume that `get_writer` is doing its job handling the parameter.

Reply:
Thanks for your comment. I got it.